Backend-Challenges.md

```diff
@@ -65,32 +80,116 @@ In `Challenge_6.t.sol`, we want to get the token URI from the contract given a c
 
 **HINT**: Solidity will not accept string and uint concatenation. You can use the provided Strings library to turn uint into a string.
 
-### Solidity libraries and openzeppelin
+### TokenURI
 
-In Solidity, you can import libraries that will make life easier. OpenZeppelin is a company that provides free standardized contract libraries to use, such as ERC721. These contracts are firmly audited, meaning they are generally secure to implement in your contracts.
+If you are familiar with [Opensea](https://opensea.io/), you might be wondering: how on earth do they get the NFT images and metadata? Well, often times these images are stored on chain, on IPFS, or even in the cloud. As on-chain storage is expensive, images are more often than not stored off-chain.
```

Comment:
I think you still need to merge the feedback I posted here from main/the other branch.